src/firefox/index.ts

```diff
@@ -283,8 +283,7 @@ class Firefox {
             error: false,
           } as UpdateLog),
         })
-        await this.stop()
```

Comment:
Ideally, we first check for updates and then only launch any of our resources. The reason for calling this was to make sure that if in case any processes are running, then we first close them before updating to avoid any problems.